### PR TITLE
MTP-1958 Newsletter checkbox issues

### PR DESCRIPTION
**Pull request by petterkoding**
Checkboxes getting squashed by long texts, MTP-1958

### CODE REVIEW

File: mt-kit/core/css/src/utilities/_layout.scss [hunk 1]
```diff
@@ -106,6 +106,7 @@
   display: flex;
   flex-direction: column;
   gap: var(--gap);
+  flex-wrap: nowrap;
```
Comment (lha047) [inline mt-kit/core/css/src/utilities/_layout.scss:109]: This is going to change it for all the layout-flex-col. Did you check that it doesnt break any thing? Suggest you make a new class with this addition

File: mt-kit/core/css/src/components/_radio-checkbox.scss [hunk 2]
```diff
@@ -25,6 +25,7 @@
   float: left;
   border-radius: 50%;
   color: var(--color-primary);
+  flex-shrink: 0;
```
Comment (lha047) [inline mt-kit/core/css/src/components/_radio-checkbox.scss:28]: Suggest you make a class for this in layout and use that. This class should not consider it self with layout based on parent

**Review by lha047 — CHANGES_REQUESTED** [review]
See comments :)

File: mt-kit/core/css/src/components/_radio-checkbox.scss (hunk 2 above)
Comment (petterkoding) [reply mt-kit/core/css/src/components/_radio-checkbox.scss]: appears that its no longer needed after using  `flex-wrap: nowrap` on the parent fieldset and on the nested fieldsets as well.

**Review by lha047 — CHANGES_REQUESTED** [review]
Screenshot 2024-03-06 at 07 53 40
Screenshot 2024-03-06 at 07 53 30

Still breaks. Update stories to show how it breaks with long sentences in checkbox labels
Look into how the `.form-control` works with the grid. It should work with the correct values. You should not need the flex no-wrap. Check the html classes on checkboxwith subsets. Look into the displays in _checkboxes-with-subsets

**Review by amundlindstol — APPROVED** [review]
LGTM :D

**Pull request merged**